frontend/src/components/episode/episode.tsx

```diff
@@ -160,7 +160,9 @@ const Episode: React.FC = () => {
           </section>
         </>
       ) : (
-        <h1 className={styles.notFound}>Oops. There is no such episode</h1>
+        dataStatus === DataStatus.REJECTED && (
```

Comment:
```suggestion
        (
```
Actually, we don't want to change this logic
we should do something with `IDLE` status